### DISCUSSION TITLE
Beatmap Info Display

### DISCUSSION THREAD

**Discussion by 19peytonsmith** [ppy/osu#19468] [Ideas] [answered]
Show beatmap stats like AR, CS, OD, HP on the main page of the beatmap when selected. These stats aren't immediately obvious, as I think they should be, rather you have to right click and go to details to find the info which is a bit cumbersome.

**Comment by frenzibyte** [comment, marked as answer]
This will be resolved in the new design of the song select screen, which would display both the details of the beatmap in an explicit fashion and the leaderboards below it:

<img width="573" alt="CleanShot 2022-07-30 at 10 12 35@2x" src="https://user-images.githubusercontent.com/22781491/181879366-be1c6897-ecba-4c5a-852c-5e9f6d23529d.png">

Note that nothing is final here, before you start making notes on the design.

**Reply by 19peytonsmith** [reply]
Oh very nice, a quick note that maybe this seems a bit too much at a first glance? Maybe have a collapsable window or show more info upon hover is more ideal to removing some of the clutter

**Reply by peppy** [reply]
It is intended to be collapsible, yes.

**Reply by Givikap120** [reply]
Just want to give a random opinion on this.
By far the most important things (besides the name) are star rating, bpm, length, and difficulty attributes
Everything else should be expandable because it's too much visual noise if not

Also, I think the strain graph from the star rating system will fit better than a density
This is discussable but instead of the one-star rating number I would like to see "difficulty to play" (kinda like the live), "difficulty to FC" (directly correlated with pp), and skills contribution (like 60% aim, 30% speed, 10% reading)

Also, one more suggestion on design - make more colors depending on the difficulty. I really like the feeling when elements become dark when you're clicking on the 9 star map.
Example - switch between diffs of this map in lazer. https://osu.ppy.sh/beatmapsets/1056699#osu/2211017
